nf/conf/kids_first.config

```diff
@@ -0,0 +1,49 @@
+process {
+    withName: SENTIEON_DNASCOPE {
+        ext.export_args = { [
+            "export SENTIEON_LICENSE=${params.sentieon_license_file ?: params.sentieon_license_server};",
```

Comment:
This approach will not work with the `license_file` on an HPC. You need to pass this file in somehow.